### PR TITLE
ER-178 - Allow more precise floor area

### PR DESCRIPTION
**Pull request by jmarrec**
## Pull Request Description

Add a new parameter "Geometry Actual Floor Area" that sets the geometry_unif_cfa to 100sqft increments

## Checklist

Not all may apply:

- [ ] Tests (and test files) have been updated
- [ ] Documentation has been updated
- [ ] Changelog has been updated
- [ ] `openstudio tasks.rb update_measures` has been run
- [ ] No unexpected regression test changes on CI (checked comparison artifacts)

### CODE REVIEW

File: measures/ResStockArguments/measure.rb [hunk 1]
```diff
@@ -401,7 +401,7 @@ def run(model, runner, user_arguments)
                ['2500-2999', HPXML::ResidentialTypeApartment] => 2648, # AHS 2021, multi-family weighted average
                ['3000-3999', HPXML::ResidentialTypeSFD] => 3310, # AHS 2021, 1 detached and mobile home weighted average
                ['3000-3999', HPXML::ResidentialTypeSFA] => 3228, # AHS 2021, 1 attached
-               ['3000-3999', HPXML::ResidentialTypeApartment] => 33171, # AHS 2021, multi-family weighted average
+               ['3000-3999', HPXML::ResidentialTypeApartment] => 3171, # AHS 2021, multi-family weighted average
```
Comment (jason-curtis) [inline measures/ResStockArguments/measure.rb:404]: nice catch :open_mouth:

**Review by jason-curtis — APPROVED** [review]
wonder what they'll think of this upstream!

File: measures/ResStockArguments/measure.rb (hunk 1 above)
Comment (jmarrec) [reply measures/ResStockArguments/measure.rb:404]: They already fixed that typo on nrel's develop fyi

**Comment by jmarrec** [comment]
> wonder what they'll think of this upstream!

Wasn't planning on upstreaming. I am almost certain this isn't of interest to them considering how resstock is meant to be used. Do you think otherwise?

**Comment by jason-curtis** [comment]
> > wonder what they'll think of this upstream!
> 
> Wasn't planning on upstreaming. I am almost certain this isn't of interest to them considering how resstock is meant to be used. Do you think otherwise?

I was assuming that resstock wants us to upstream it based on Balwinder's comment here: https://nrgx-modeling.atlassian.net/browse/ER-178?focusedCommentId=10864 but I may be misunderstanding it.

image

**Comment by jmarrec** [comment]
Pretty sure that was "Requires", not "Request".

**Pull request merged**